packages/frontend/src/components/Lp/MintAndLp/DepositPreviewModal.tsx

```diff
@@ -0,0 +1,222 @@
+import React from 'react'
+import { Modal, Box, Typography, Stepper, Step, StepLabel } from '@material-ui/core'
+import { makeStyles, createStyles } from '@material-ui/core/styles'
+import Image from 'next/image'
+import clsx from 'clsx'
+
+import { AltPrimaryButton } from '@components/Button'
+import walletIcon from 'public/images/wallet.svg'
+import LpSettings from './LpSettings'
+
+const useWaitForConfirmationStyles = makeStyles((theme) =>
+  createStyles({
+    iconWrapper: {
+      backgroundColor: theme.palette.background.lightStone,
+      borderRadius: '100%',
+      padding: theme.spacing(2.5),
+    },
+    icon: {
+      height: '30px',
+      width: '30px',
+    },
+    title: {
+      fontSize: '22px',
+      fontWeight: 700,
+      letterSpacing: '-0.01em',
+    },
+  }),
+)
+
+const WaitForConfirmation: React.FC = () => {
+  const classes = useWaitForConfirmationStyles()
+
+  return (
+    <Box display="flex" flexDirection="column" alignItems="center" gridGap="48px">
+      <div className={classes.iconWrapper}>
+        <div className={classes.icon}>
+          <Image src={walletIcon} alt="wallet" />
+        </div>
+      </div>
+
+      <Typography className={classes.title}>Confirm transaction in your wallet</Typography>
+    </Box>
+  )
+}
+
+const useStepperStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      backgroundColor: 'inherit',
+      padding: theme.spacing(5, 0),
+
+      '& [class|="MuiStepConnector-root"]': {
+        padding: 0,
+      },
+
+      '& [class|="MuiStepConnector-line"]': {
+        minHeight: '40px',
+      },
+    },
+  }),
+)
+
+function getSteps() {
+  return ['Depositing ETH', 'Earning interest']
```

Comment:
lets actually just make this one step of 'Depositing ETH' since the user isn't earning fees + funding at their deposit, they earn fees + funding over time

Reply:
Got it. If not "Earning interest", there should be something else I think. Otherwise the stepper design looks incomplete.

<img width="1401" alt="Screenshot 2022-11-21 at 7 16 45 PM" src="https://user-images.githubusercontent.com/12045121/203070883-3455c95e-7ddb-4dec-8b02-fc2d10b5850b.png">

---

For now I'm removing the stepper component and just keeping "view dashboard" there.

<img width="1388" alt="Screenshot 2022-11-21 at 7 15 33 PM" src="https://user-images.githubusercontent.com/12045121/203070657-4103ae31-3755-4ea3-9df8-42dc35ba987e.png">